pkg/ipcache/types/types.go

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package ipcachetypes
+
+import (
+	"context"
+	"sync"
+
+	"github.com/cilium/cilium/pkg/identity/cache"
+)
+
+type PolicyHandler interface {
```

Comment:
Can we get some comments for both these interfaces? Especially around any rules or assumptions implementations may need to be aware of.